### PR TITLE
docs: added example for awscc_neptune_db_cluster

### PR DESCRIPTION
**Pull request by quixoticmonk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

Blocked by #1735

### CODE REVIEW

File: examples/resources/awscc_neptune_db_cluster/neptune_db_cluster.tf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  backup_retention_period        = 5
+  preferred_backup_window        = "07:00-09:00"
+  iam_auth_enabled               = true
+  db_port                        = 8182
+  copy_tags_to_snapshot          = true
+  enable_cloudwatch_logs_exports = ["audit"]
+  engine_version                 = "1.3.1.0"
+  preferred_maintenance_window   = "sun:10:00-sun:10:30"
+  storage_encrypted              = true
+  kms_key_id                     = awscc_kms_key.example.key_id
```
Comment (wellsiau-aws) [inline examples/resources/awscc_neptune_db_cluster/neptune_db_cluster.tf:12]:
```suggestion
  kms_key_id                     = awscc_kms_key.example.arn
```

I suggest to use arn to address issue reported at #1735

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
just a minor update requested, thank you!

**Review by wellsiau-aws — APPROVED** [review]
LGTM

**Comment by marcosentino** [comment]
@quixoticmonk Thank you for the contribution!

**Pull request merged**